### PR TITLE
upgrading finagle version to 6.35.0

### PR DESCRIPTION
**Pull request by revathy144**
Upgrading finagle version to 6.35.0. We use tranquility for druid in our organization **Tapad**.  Tranquility uses scala-utils as a dependency. Scala-utils uses 6.31.0 version of finagle which is not compatible to some of our organization's services. So we request you to upgrade finagle's version to 6.35.0.

### CODE REVIEW

File: src/main/scala/com/metamx/common/scala/net/finagle/DiscoResolver.scala [hunk 1]
```diff
@@ -3,12 +3,14 @@ package com.metamx.common.scala.net.finagle
 import com.metamx.common.lifecycle.Lifecycle
 import com.metamx.common.scala.Logging
 import com.metamx.common.scala.net.curator.Disco
-import com.twitter.finagle.{Addr, Resolver}
+import com.twitter.finagle.{Addr, Address, Resolver}
 import com.twitter.util.{Closable, Future, Time, Var}
 import java.net.{InetSocketAddress, SocketAddress}
+
```
Comment (logarithm) [inline src/main/scala/com/metamx/common/scala/net/finagle/DiscoResolver.scala]: Unnecessary empty line

File: src/main/scala/com/metamx/common/scala/net/finagle/DiscoResolver.scala [hunk 2]
```diff
@@ -3,12 +3,14 @@ package com.metamx.common.scala.net.finagle
...
 import org.apache.curator.framework.CuratorFramework
 import org.apache.curator.framework.state.ConnectionState
 import org.apache.curator.x.discovery.details.ServiceCacheListener
+
```
Comment (logarithm) [inline src/main/scala/com/metamx/common/scala/net/finagle/DiscoResolver.scala]: Unnecessary empty line

File: src/main/scala/com/metamx/common/scala/net/finagle/DiscoResolver.scala [hunk 3]
```diff
@@ -3,12 +3,14 @@ package com.metamx.common.scala.net.finagle
 import com.metamx.common.lifecycle.Lifecycle
 import com.metamx.common.scala.Logging
 import com.metamx.common.scala.net.curator.Disco
-import com.twitter.finagle.{Addr, Resolver}
+import com.twitter.finagle.{Addr, Address, Resolver}
 import com.twitter.util.{Closable, Future, Time, Var}
 import java.net.{InetSocketAddress, SocketAddress}
```
Comment (logarithm) [inline src/main/scala/com/metamx/common/scala/net/finagle/DiscoResolver.scala]: `SocketAddress` is not used anymore, so it might be removed

**Comment by logarithm** [comment]
@revathy144 Thank you for your contribution! I just want to inform you that we are going to update `scala-util` dependencies to the latest finagle versions due to create build for scala 2.12. I hope we will release it soon. But until then you can exclude finagle for scala-util and tranquility in your project.

**Comment by cheleb** [comment]
@logarithm any news of scala 2.12 support ?

**Comment by logarithm** [comment]
@cheleb thanks for heads up! Yes, I've already migrated scala-util to scala 2.12, but I haven't pushed it yet because I'm waiting for our other public repos to be released and updated first. I hope in the next week or two we will release scala 2.12 support.

**Comment by cheleb** [comment]
@logarithm Cool, I'm waiting this (and tranquility-core) to deploy akka-stream sink with alpakka !

**Comment by logarithm** [comment]
@cheleb @revathy144 I'm happy to inform you, that https://github.com/metamx/scala-util/pull/27 has been merged into master branch and new version of scala-util 0.13.0 has been released. As far as I understand this PR might be closed.

**Comment by logarithm** [comment]
@revathy144 Could you please close this PR since requested changes were merged in separate PR. Thanks!

**Pull request closed without merging**